week-3/D-array-filter/exercise.js

```diff
@@ -8,7 +8,15 @@
 
 var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-var pairsByIndex; // Complete this statement
+var pairsByIndex = pairsByIndexRaw.filter(isArray);
+
+function isArray(item) {
+  if (item !== null && item.length > 1) {
```

Comment:
I think you're looking for length of exactly 2. Also, you're returning `Array.isArray(item)` if true and otherwise, just calling it - think that can be improved :)